Dockerfile

```diff
@@ -0,0 +1,14 @@
+# run with:
+# docker build -t lazygit
```

Comment:
this command lacks a directory, a single dot should be added at the end if we assume one is in lazygit directory already with Dockerfile in it, so:
`docker build -t lazygit .`